lib/SourceMapSource.js

```diff
@@ -8,6 +8,9 @@ const Source = require("./Source");
 const streamChunksOfSourceMap = require("./helpers/streamChunksOfSourceMap");
 const streamChunksOfCombinedSourceMap = require("./helpers/streamChunksOfCombinedSourceMap");
 const { getMap, getSourceAndMap } = require("./helpers/getFromStreamChunks");
+const {
+	isDualStringBufferCachingEnabled
+} = require("./helpers/stringBufferUtils");
 
 class SourceMapSource extends Source {
 	constructor(
```

Comment:
I think we want to internal `value` and  `originalSource` here?

Reply:
Same comment as https://github.com/webpack/webpack-sources/pull/155/files#r1654650118.